components/table/index.ts

```diff
@@ -1,6 +1,3 @@
 import Table from './table'
-import TableColumn from './table-column'
-
-Table.Column = TableColumn
```

Comment:
why remove this and move to table.ts

Reply:
Less code, no need to write import RcTable twice